promql/promql-test-queries.yml

```diff
@@ -45,13 +45,22 @@ test_cases:
   # TODO: Add tests for staleness support.
   - query: 'demo_memory_usage_bytes'
   - query: '{__name__="demo_memory_usage_bytes"}'
+  # Check for spacing.
+  - query: ' { __name__ = "demo_memory_usage_bytes" } '
```

Comment:
Maybe put this into the lower section where we have multiple matchers, and have spaces around the comma as well?